man/TestLinear.Rd

```diff
@@ -27,8 +27,11 @@ Defaults to TRUE.}
 \item{leveling_off}{If TRUE, uses post and overidpost from estimates to test for leveling-off.
 Defaults to TRUE.}
 }
+\value{
+A data frame containing the F-statistic and p-value for the specified test(s).
+}
 \description{
-This function can be used to perform tests of linear hypotheses.
+\code{TestLinear} tests whether a variable is equal to a given quantity or to another variable.
```

Comment:
```suggestion
\code{TestLinear} tests linear restrictions on coefficients.
```

Reply:
@jmshapir the way to edit `man/FunctionName.Rd` files is (surprisingly) by editing the corresponding `FunctionName.R` file and then running [devtools::document()](https://r-pkgs.org/whole-game.html#sec-whole-game-document).

I did this in https://github.com/JMSLab/eventstudyr/pull/32/commits/ca4e029520fa3ba0a18117b4d3fae837d0339317 to implement your above commit suggestion.